Userland/Libraries/LibVideo/CodecID.h

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2023, Stephan Vedder <stephan.vedder@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/HashFunctions.h>
+#include <AK/Time.h>
+#include <AK/Traits.h>
+#include <AK/Types.h>
+#include <AK/Variant.h>
+
+namespace Video {
+
+enum class CodecID : u32 {
+    Unknown,
+    // On2 / Google
+    VP8,
+    VP9,
+    // MPEG
+    H261,
+    H262,
+    H263,
+    H264,
+    H265,
+    MPEG1,
+    MPEG2 = H262,
+    MPEG4 = H263,
```

Comment:
Is this a typo that should be H264?
MPEG4 isn't the name of a codec anyway (only MPEG-4 Part 10), so I'd suggest this change:
```suggestion
    AVC = H264,
    HEVC = H265,
```